sources/packages/web/src/components/aest/students/assessment/History.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-container>
+        <p class="category-header-large color-blue">
```

Comment:
Can we please use the BodyHeader component here with the count also, like below:
```html
<body-header
      title="Designation agreements"
      :recordsCount="designations?.length"
      class="m-1"
>
```